server/http/src/server.rs

```diff
@@ -1,22 +1,17 @@
-use futures::future;
-use futures::prelude::*;
 use futures::sync::mpsc::{channel, Receiver, Sender};
 use hyper;
 use hyper::Server;
 
-use slog;
 use std::error::Error;
 use std::fmt;
 use std::net::{Ipv4Addr, SocketAddrV4};
-use std::sync::{Arc, Mutex};
+use std::sync::Mutex;
```

Comment:
Why is `Arc` no longer needed here? included in prelude?

Reply:
exactly